README.md

```diff
@@ -81,6 +81,9 @@ See also:
 
 #### Progressive Web Apps (PWAs)
 
+<!-- todo: add https://github.com/captainbrosset/movies-db-pwa/ -->
```

Comment:
Move https://github.com/captainbrosset/movies-db-pwa/ to Demos repo
@captainbrosset